src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -71,6 +71,28 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
                         If Not CaseInsensitiveComparison.Equals(valueText, idText) Then
                             Return newNode.ReplaceToken(literal, CreateDecimalLiteralToken(literal, valueText, value))
                         End If
+                    Case SyntaxKind.IntegerLiteralToken
+                        ' Get the literal identifier text which needs to be pretty listed.
+                        Dim idText = literal.GetIdentifierText()
+                        Dim value As Long = CType(literal.Value, Long)
+
+                        If value = 0 Then
+                            ' Overflow/underflow case or zero literal, skip pretty listing.
```

Comment:
Does using Long work for UInteger at the extremes. Actually use ULong. There shouldn't be any negative integer tokens, unless it's in hex.